config/routes.rb

```diff
@@ -25,6 +25,7 @@
     resources :proposed_organisation_edits, :only => [:new, :show, :create, :update]
   end
   resources :users
+  resources :events
```

Comment:
Do you intend to implement all the RESTful actions ? becaause from `events_controllerrb` there is only `new `and `create` implemented 
For best practices tend to add the routes that you only need to help developers understand what `events_controller `is doing by looking at the available routes, for this example:
`resources :events, only: [:new, :create]`